common.sh

```diff
@@ -164,9 +164,40 @@ die_notrace() {
   for line in "$@"; do
     error "${DIE_PREFIX}${line}"
   done
+  if [[ ! -e "${SCRIPTS_DIR}/NO_DEBUG_OUTPUT_DELETE_ME" ]]; then
+      error "${DIE_PREFIX}!!!!!!!!!!!!!!!!!!!!!!!!!"
+      error "${DIE_PREFIX}!! BEGIN DEBUG OUTPUT: !!"
+      error "${DIE_PREFIX}!!!!!!!!!!!!!!!!!!!!!!!!!"
+      error
+      error "${DIE_PREFIX}== MOUNT =="
+      error "${DIE_PREFIX}==========="
+      error_command_output "${DIE_PREFIX}" mount
+      error
+      error "${DIE_PREFIX}== DF =="
+      error "${DIE_PREFIX}==========="
```

Comment:
```suggestion
      error "${DIE_PREFIX}========"
```

I'll fix it after github action is done.